content/maxwell3_fdem/MT/MT_N_layered_Earth.rst

```diff
@@ -0,0 +1,329 @@
+-------------------
+
+The governing equation for MagnetoTellurics problem can be obtained from
+:ref:`Maxwell's equations <maxwell1_fundamentals_index>`. We start with :ref:`Faraday's law<faraday>` and :ref:`Ampere's law<ampere_maxwell>`:
+
+.. math::
+    \nabla \times \mathbf{E_x} = - i \omega \mu \mathbf{H_y}
+    :label: Faraday
+    
+.. math::
+    \nabla \times \mathbf{H_y} = (\sigma + i \omega \varepsilon) \mathbf{E_x}
+    :label: Ampere
+
+Knowing that the divergent of **E** and **H** are equal to 0 here (no free charge)
+according to :ref:`Gauss's Law for Electric Fields<gauss_electric>` and :ref:`Gauss's Law for Magnetic Fields<gauss_magnetic_frequency>`, we can combine the equations to write the Helmhotz (wave propagation) equation for both **E** and **B** field:
+
+.. math::
+    \nabla ^2  \mathbf{E_x} + k^2 \mathbf{E_x} = 0
+    :label: E_wave_propagation_equation
+
+.. math::
+    \nabla ^2 \mathbf{H_y} + k^2 \mathbf{H_y} = 0
+    :label: H_wave_propagation_equation
+
+with k the wavenumber:
+
+.. math::
+    k = \sqrt{\omega ^2 \mu \varepsilon - i \omega \mu \sigma }
+    :label: kwavenumber
+
+
+In the ground, we usually assume that the displacement current is negligible, which means :math:`\sigma >> \omega \varepsilon`. In this case 
+
+.. math::
+    k_{ground} \simeq (1-i) \sqrt{ \frac{\omega \mu \sigma}{2} }
+    :label: kwavenumber_steadystate
+
+In the air, the conductivity is almost 0
+
+.. math::
+    k_{air} \simeq \omega \sqrt{ \mu_0 \varepsilon_0}
+    :label: kwavenumber_air
+
+Taking the problem from the point of view of the electric field, we know the equation :eq:`E_wave_propagation_equation` has a solution in the form of:
+
+.. math::
+    E_x (z) = U e^{i k z} + D e^{-i k z}
+    :label: Electric field components
+    
+.. math::
+    H_y (z) = \frac{(\nabla \times \mathbf{E_x})_y}{- i \omega \mu} = \frac{k}{ \omega \mu} (D e^{-i k z} - U e^{i k z}) = \frac{1}{Z} (D e^{-i k z} - U e^{i k z})
+    :label: Magnetic field components
+
+with :math:`\mathbf{E_x} = E_x \mathbf{\hat{x}}`  and U and D are the complex amplitudes of the Up and Down components of the field and Z the intrinsic impedance of the space.
+
+Writing the solution in the j-th layer (See :numref:`MTlayeredEarth`), we got:
+
+ .. math::
+    E_{x,j} (z) = U_j e^{i k (z-z_{j-1})} + D_j e^{-i k (z-z_{j-1})}
+    :label: Electric field components in layers
+    
+ .. math::
+    H_{y,j} (z) = \frac{1}{Z_j} (D_j e^{-i k (z-z_{j-1})} - U_j e^{i k (z-z_{j-1})})
+    :label: Magnetic field components in layers
+
+
+Which can be rewrite as:
+
+ .. math::
+    \left(\begin{matrix} E_{x,j} \\ H_{y,j} \end{matrix} \right) = \left(\begin{matrix} 1 & 1 \\ -\frac{1}{Z_j} & \frac{1}{Z_j} \end{matrix} \right) \left(\begin{matrix} U_j \\ D_j \end{matrix} \right) 
+    = P_j \left(\begin{matrix} U_j \\ D_j \end{matrix} \right)
+    :label: Propagation matrix 
+
+The transition of the Up and Down component inside a layer can then be write as such
```

Comment:
what do you mean by "transition"? is it maybe "the propagation of the Up and Down going components within a layer ..." ?